### ISSUE TITLE
Compilation error with Unity 5.0 on Windows Universal App 8.1

### ISSUE THREAD

**Issue by LateNightApps** [TouchScript/TouchScript#176]
I recently added TouchScript to my Unity project which up until then was building and deploying correctly. After I added TouchScript I am suddenly getting this error:

Error	11	The command ""C:\Program Files\Unity\Editor\Data\PlaybackEngines\metrosupport\Tools\AssemblyConverter.exe" -platform=wsa81 "D:\Projects\Words3\Words3Unity\bin\Store 8.1\x86\Master\Assembly-CSharp.dll" "D:\Projects\Words3\Words3Unity\bin\Store 8.1\x86\Master\Assembly-CSharp-firstpass.dll" "D:\Projects\Words3\Words3Unity\Builds\Windows Universal\Words3\Words3.Windows\\UnityEngine.dll" "D:\Projects\Words3\Words3Unity\Builds\Windows Universal\Words3\Words3.Windows\\WinRTLegacy.dll" "D:\Projects\Words3\Words3Unity\Builds\Windows Universal\Words3\Words3.Windows\\UnityEngine.UI.dll"" exited with code 1.	

and a similar error for the Windows Phone build. I have grabbed the latest TouchScript unity package from here, imported in into Unity, rebuilt the project and still it gives me the error. I am thinking that I am doing something wrong with Unprocessed dll's but I am relatively new to adding external packages. 

Has anyone see this with your package before or do you have any suggestions?

Thanks

**Comment by valyard** [comment]
Have you tried with an empty project or a TouchScript example?
Which version are you using?

**Comment by LateNightApps** [comment]
I tried it with the version downloaded from the asset store and then I also tried with version 6.6 downloaded from here. I will go and try to make an empty project to see if that still shows the problem.

**Comment by LateNightApps** [comment]
The problem didn't present itself in a clean project so I analyzed the .csproj files that were generated and found a couple differences. I should have realized that because Unity wasn't regenerating the .csproj files that I would have to manually add in any external .dlls. I had to modify the BeforeResolveReferences section to copy down the TouchScript.dll/.pdb and the BeforeCompile to include the TouchScript.dll in the AssemblyConverter.
